### ISSUE TITLE
Add HeatExchanger:AirToAir:SensibleAndLatent as a valid air loop component

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5156]
Helpdesk ticket 10728
HeatExchanger:AirToAir:SensibleAndLatent on an airloop branch results in in a fatal error.

```
** Severe ** GetAirPathData: AirLoopHVAC = "AHU". 
** ~~~ ** ..Invalid Air Loop Component Type = "HEATEXCHANGER:AIRTOAIR:SENSIBLEANDLATENT". 
** ~~~ ** ..Air Loop Component Name = "ENTHALPY HX". 
** ~~~ ** ..reference Branch = "AHU MAIN BRANCH". 
** Fatal ** GetAirPathData: Errors found retrieving input for AirLoopHVAC. 
```

Looking at the [code that processes the air loop components](https://github.com/NREL/EnergyPlus/blob/bc463b16f9ecaecd6c9ace48676400f05e0aa64f/src/EnergyPlus/SimAirServingZones.cc#L1168-L1173), HEATEXCHANGER:DESICCANT:BALANCEDFLOW and HEATEXCHANGER:AIRTOAIR:FLATPLATE are valid.  I don't see any reason why HeatExchanger:AirToAir:SensibleAndLatent should not be valid also.  @rraustad  ?  

This was probably just an omission when the HeatExchanger:AirToAir:SensibleAndLatent was added as a new component.

**Comment by rraustad** [comment]
Well, back then, we simply tried to figure out all the places a component would need to be added to the code. It might be that flat plate was inadvertently added to the air loop component list. However, I also see no reason not to allow that configuration other than a flow ratio out of range issue for users that aren't careful. I question the ability to add this component to the air loop main branch and the OASys equipment list since a component type and name must be included in the OA system equipment list. These would be the same name and type used by the air loop equipment list. I think I tried that before but can't recall if it failed for this reason or something else.

**Comment by Nigusse** [comment]
@mjwitte @rraustad I added "HEATEXCHANGER:AIRTOAIR:SENSIBLEANDLATENT" as a valid component.  The defect file run completed successfully.  However, I see a large number of "unbalanced air volume flow ratio through the heat exchanger is greater than 2:1" error message.

**Comment by mjwitte** [comment]
@Nigusse Is the HX flow rate autosized?  The sizing routine may need to be sensitive to the location of the HX?

**Comment by Nigusse** [comment]
@mjwitte It was hardsized. I re-run with autosized HX flow rate.  The number of recurring error messages did not change much.

**Comment by mjwitte** [comment]
@Nigusse Looking at the file now - since the HX is being added to a VAV system, the warnings are to be expected.  Adding some more logic and/or inputs to the heat exchanger model to cover a wider range of flow rates would be useful, but that's a different issue.  For this one, the key would be to check that the heat exchanger is actually being simulated and producing leaving conditions that make sense.

**Comment by Nigusse** [comment]
@mjwitte The heat exchanger on the main branch is simulated, the air outlet conditions are as expected.  The CI machine also came clean, I will add a unit test to complete this fix.

**Comment by mjwitte** [comment]
Closed via #5392
